include/drivers/gpio.h

```diff
@@ -1158,6 +1286,9 @@ static inline int gpio_remove_callback(struct device *port,
  * Note: Depending on the driver implementation, this function will enable
  *       the pin to trigger an interruption. So as a semantic detail, if no
  *       callback is registered, of course none will be called.
+ *
+ * @deprecated Replace with ``gpio_pin_interrupt_configure()`` with
+ * ``GPIO_INT_ENABLE``.
```

Comment:
The exact same text seems to have been present in my original PR before it was removed. But it is outdated. When `gpio_pin_interrupt_configure()` function is called it needs to be passed a full specification of the interrupt it needs to configure. Passing `GPIO_INT_ENABLE` only is not enough.

Reply:
done